bundles/org.openhab.binding.ferroamp/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,538 @@
+				<label>Grid Energy Produced L3</label>
+			</channel>
+			<channel id="grid-energy-consumed-l1" typeId="energy">
+				<label>Grid Energy Consumed L1</label>
+			</channel>
+			<channel id="grid-energy-consumed-l2" typeId="energy">
+				<label>Grid Energy Consumed L2</label>
+			</channel>
+			<channel id="grid-energy-consumed-l3" typeId="energy">
+				<label>Grid Energy Consumed L3</label>
+			</channel>
+			<channel id="inverter-energy-produced-l1" typeId="energy">
+				<label>Inverter Energy Produced L1</label>
+			</channel>
+			<channel id="inverter-energy-produced-l2" typeId="energy">
+				<label>Inverter Energy Produced L2</label>
+			</channel>
+			<channel id="inverter-energy-produced-l3" typeId="energy">
+				<label>Inverter Energy Produced L3</label>
+			</channel>
+			<channel id="inverter-energy-consumed-l1" typeId="energy">
+				<label>Inverter Energy Consumed L1</label>
+			</channel>
+			<channel id="inverter-energy-consumed-l2" typeId="energy">
+				<label>Inverter Energy Consumed L2</label>
+			</channel>
+			<channel id="inverter-energy-consumed-l3" typeId="energy">
+				<label>Inverter Energy Consumed L3</label>
+			</channel>
+			<channel id="load-energy-produced-l1" typeId="energy">
+				<label>Load Energy Produced L1</label>
+			</channel>
+			<channel id="load-energy-produced-l2" typeId="energy">
+				<label>Load Energy Produced L2</label>
+			</channel>
+			<channel id="load-energy-produced-l3" typeId="energy">
+				<label>Load Energy Produced L3</label>
+			</channel>
+			<channel id="load-energy-consumed-l1" typeId="energy">
+				<label>Load Energy Consumed L1</label>
+			</channel>
+			<channel id="load-energy-consumed-l2" typeId="energy">
+				<label>Load Energy Consumed L2</label>
+			</channel>
+			<channel id="load-energy-consumed-l3" typeId="energy">
+				<label>Load Energy Consumed L3</label>
+			</channel>
+			<channel id="grid-energy-produced-total" typeId="energy">
+				<label>Grid Energy Produced Total</label>
+			</channel>
+			<channel id="grid-energy-consumed-total" typeId="energy">
+				<label>Grid Energy Consumed Total</label>
+			</channel>
+			<channel id="inverter-energy-produced-total" typeId="energy">
+				<label>Inverter Energy Produced Total</label>
+			</channel>
+			<channel id="inverter-energy-consumed-total" typeId="energy">
+				<label>Inverter Energy Consumed Total</label>
+			</channel>
+			<channel id="load-energy-produced-total" typeId="energy">
+				<label>Load Energy Produced Total</label>
+			</channel>
+			<channel id="load-energy-consumed-total" typeId="energy">
+				<label>Load Energy Consumed Total</label>
+			</channel>
+			<channel id="total-solar-energy" typeId="power">
+				<label>Total Solar Energy</label>
+			</channel>
+			<channel id="state" typeId="string">
+				<label>State of the System</label>
+			</channel>
+			<channel id="timestamp" typeId="date-time">
+				<label>Time Stamp When Message was Published</label>
```

Comment:
Please check all channel labels to have 2-3 words, < 23 characters and each word is capitalized
```suggestion
				<label>Published Time</label>
```